tools/sds-aspect-model-maven-plugin/src/main/java/io/openmanufacturing/sds/aspectmodel/AspectModelMojo.java

```diff
@@ -93,12 +95,12 @@ private void handleFailedModelResolution( final Try<VersionedModel> failedModel
 
       // Another exception, e.g. syntax error. Let the validator handle this
       final AspectModelValidator validator = new AspectModelValidator();
-      final ValidationReport report = validator.validate( failedModel );
-      throw new MojoExecutionException( report.toString(), loadModelFailureCause );
+      final List<Violation> violations = validator.validateModel( failedModel );
```

Comment:
IMHO the option to print the detailed output for any possible violations should be available here as well.

Reply:
Done